.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+#### What type of PR is this?
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind failing-test
+/kind flake
+
+Optionally add one of the following areas, help us further classify and filter PRs:
+/area scheduling
+/area controllers
+/area cli
+/area dependency
+/area webhooks
+/area deploy
+/area documentation
+/area performance
+/area test
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
```

Comment:
I think we can add the notice that do not add `Fixes #` if the issue has associated with multiple PRs.

Reply:
updated